addons/godot_rl_agents/sync.gd

```diff
@@ -1,11 +1,21 @@
 extends Node
+class_name Sync
```

Comment:
Somehow I had to add a class name to make the descriptions for ControlModes show up in the editor.

Reply:
I'm also not seeing the suggestions with Godot 4.3 beta 2 even with the class_name, I will test it more, maybe removing `.godot` folder will help after updating the plugin.

![image](https://github.com/edbeeching/godot_rl_agents_plugin/assets/61947090/fc10f12d-5bd0-4499-942e-d9c8859efea4)